src/styles/_catalog.scss

```diff
@@ -14,9 +14,15 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: transform 300ms, color 300ms, ;
```

Comment:
I think it could be shortened not to use value twice. Like 
transition: 300ms;
It worked for my catalog at least ;)